Set default content and fix lint issues (#28228)
TheOdinProject/curriculum: ruby/basic_ruby/hashes.md

> For a full list of the methods that work on hashes, check out the [Ruby Docs](https://docs.ruby-lang.org/en/3.3/Hash.html)

Reviewer: More descriptive link while we're at it
```suggestion
For a full list of the methods that work on hashes, check out the [Hash class documentation](https://docs.ruby-lang.org/en/3.3/Hash.html).
```

<div class="lesson-content__panel" markdown="1">

> 1. Read Launch School's chapter on [Hashes](https://launchschool.com/books/ruby/read/hashes), and go through the exercises using irb or any other REPL, such as [replit.com](https://replit.com/languages/ruby).

Reviewer: ```suggestion
1. Read [Launch School’s chapter on Hashes](https://launchschool.com/books/ruby/read/hashes), and go through the exercises using irb or any other REPL, such as [replit.com](https://replit.com/languages/ruby).
```

> 1. Complete the [hash](https://github.com/TheOdinProject/ruby-exercises/tree/main/ruby_basics) exercises from the [ruby-exercises repo](https://github.com/TheOdinProject/ruby-exercises) that you previously cloned.

Reviewer: We can remove the first link since we're already linking to the repo straight after
```suggestion
1. Complete the hash exercises from the [ruby-exercises repo](https://github.com/TheOdinProject/ruby-exercises) that you previously cloned.
```
